### ISSUE TITLE
Create entry for the developer documentation 

### ISSUE THREAD

**Issue by marcinczenko** [CompuIves/codesandbox-client#462]
I've just created this issue to make it visible. As discussed with @CompuIves on discord:

> It'd be very nice to have a separate section for developer documentation on the homepage
That would encourage users looking around to read it
If you are on this page [https://codesandbox.io/docs/](https://codesandbox.io/docs/) We could show 2 collapsible menus in the navigation. One called `Documentation` and one called `Developer Documentation`

I will try to have it ready soon. As an initial driver I will document some of the work that has recently been done by @gautamarora when working on issue #364 (Running Unit Tests in CodeSandbox).

**Comment by gautamarora** [comment]
@marcinczenko - amazing!! so excited for this.

**Comment by CompuIves** [comment]
This would be reaaally good to have! Thank you @marcinczenko

**Comment by marcinczenko** [comment]
Yes, it is coming very soon :).

**Comment by lbogdan** [comment]
@marcinczenko Just doing a routine spring cleanup here, are you still willing / available to work on this? I think this would be a very good encouragement for people that want to contribute, but are being put off by the apparent complexity of the project. I too plan to make an overview of how the eval manager does its magic, sometime soon.

**Comment by marcinczenko** [comment]
@lbogdan Sorry about this. I was blocked by my work. I am still willing to handle that, but I am very limited in time before 14th of May. As I agreed last time with @CompuIves the task is:

> Add Developer Documentation section on the current Docs page, but when clicking moving over to a separate page with only developer documentation.

Of course, if I happen to block someone with this, let me know.

**Comment by marcinczenko** [comment]
Sorry again for delay - I am a bit overloaded with other projects. I just created [dev-docs](https://github.com/marcinczenko/codesandbox-client/tree/dev-docs) branch today. Homepage dependencies are 3 month old, so I started with updating them to the most recent ones. It does not seem to break anything, so I will continue on top of it. I will keep you posted.

**Comment by lbogdan** [comment]
Hey @marcinczenko ,

Time for the spring cleanup 🙂 , any updates on this?

**Comment by marcinczenko** [comment]
I am sorry @lbogdan. As you see I am not dependable (I am just busy). I do not have feeling it something important to anyone at the moment. 
I am fine with closing this issue.

**Comment by SaraVieira** [comment]
Closing as we are also changing the layout of the homepage

Feel free to reopen or maybe even create a new one with what you would like to see in the `developer docs` so someone else can also pick it up

Make sense @marcinczenko ?

Thank you again!
